docs/docs/sourcing-from-gentics-mesh.md

```diff
@@ -0,0 +1,198 @@
+
+You can install this component with:
+
+```shell
+  npm install --save gatsby-source-graphql
+```
+
+### Configure the plugin
+
+The last step required before you can query your data is to configure the `gatsby-source-graphql` plugin. Open `gatsby-config.js` and add the following object to the plugins array.
+This example uses the local Gentics Mesh Demo server that you have just started.
+
+```js
+{
+    resolve: "gatsby-source-graphql",
+        options: {
+        // The top level query type, can be anything you want!
+        typeName: "MESH",
+        // The field you'll query against, can also be anything you want.
+        fieldName: "mesh",
+        // Local Gentics Mesh server url
+        url: "http://localhost:8080/api/v2/demo/graphql",
+    },
+},
+```
+
+If everything works correctly, you should now have your Gentics Mesh API added to the Gatsby source API!
+
+### Querying for content
+
+From the root of your project, run the development environment with `npm run develop`. Once the server has started, you should be able to open the following URL in your browser:
+
+`http://localhost:8000/___graphql`
+
+This will show you an interface where you can test your new content GraphQL API.
+
+Try running this query:
+
+```js
+query {
+  mesh {
+    nodes(filter: {schema: {is: vehicle}}, perPage: 2, page:5) {
+      elements {
+        uuid
+        ... on MESH_vehicle {
+          fields {
+            slug
+            weight
+            vehicleImage {
+              path
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Again, if everything is working properly, you should see a successful response in the shape of:
+
+```json
+{
+  "data": {
+    "mesh": {
+      "nodes": {
+        "elements": [
+          {
+            "uuid": "a5d81285b4884df1981285b488adf1b5",
+            "fields": {
+              "slug": "embraer-legacy-600",
+              "weight": 16000,
+              "vehicleImage": {
+                "path": "/images/embraer-legacy-600.jpg"
+              }
+            }
+          },
+          {
+            "uuid": "adaf48da8c124049af48da8c12a0493e",
+            "fields": {
+              "slug": "delorean-dmc-12",
+              "weight": 1230,
+              "vehicleImage": {
+                "path": "/images/delorean-dmc-12.jpg"
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+[You can read more about how to use the GraphQL API in the docs.](https://getmesh.io/docs/graphql/)
+
+### Getting content on the page
+
+For the purpose of this guide I've removed all the layout, SEO, link or other components in the Gatsby starter. The components are still there and 99% of users will likely want to put them back in once they understand what's happening in the code. Open the index file located at `src/pages/index.js` and replace the content with this code:
```

Comment:
Sorry, I don't follow but how is this paragraph relevant to the following parts? You're not mentioning or referencing the parts.

Reply:
I simplified the wording and referenced the file that contains the mentioned components.